qiskit_experiments/curve_analysis/utils.py

```diff
@@ -0,0 +1,47 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+A set of utility functions.
+"""
+
+from typing import Union, Optional
+
+import numpy as np
+from uncertainties.core import UFloat
+
+
+def is_error_not_significant(
```

Comment:
The way this is currently used feels like it should be an internal method of `CurveAnalysis` rather than add a new file if its not used anywhere outside of the curve analysis subclasses.